download/index.markdown

```diff
@@ -39,37 +39,70 @@ available:
 - MATE
 - XFCE
 
+These images are named `void-live-ARCH-BUILD_DATE[-FLAVOUR].iso`, where
+
+- `ARCH` is the architecture of the image, which can be:
+   - `x86_64`
+	- `x86_64-musl`
+	- `i686`
```

Comment:
I object to this sublisting here.

Reply:
In which point? I just realized the formatting got wonky (tabs vs spaces), so I can fix that. What other issues do you have with it?

Reply:
@Vaelatern I can list archs as x86_64 and i686 and then say that musl is available for x86_64. Do you think that'd be better?

Reply:
If you look at it, you see that x86_64 has a child list instead of the others being at the same level.

Reply:
nvm, this is fixed.